### PR TITLE
Multiple changes

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Submitted new location of source side validation script, and couple other changes

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md [hunk 1]
```diff
@@ -359,6 +359,9 @@ A number of commands now need to be run both in your Exchange Server on-premises
 
    > 91edc6dd-478a-497c-8731-b0b793f5a986
 
+   > [!NOTE]
+The PF mailbox GUID mentioned in prvious command must be obtained from On-Premises server, if it is obtained from Exchange Online, the migration batch will fail with error "Cannot find a recipient that has mailbox GUID"
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md:363]:
```suggestion
The public folder mailbox GUID mentioned in the previous command must be obtained from the on-premises server; if it is obtained from Exchange Online, the migration batch will fail with the error "Cannot find a recipient that has mailbox GUID".
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**